### PR TITLE
Update readme.

### PR DESCRIPTION
**Pull request by qudongfang**
Update readme.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,81 +3,85 @@
...
 
-Official git repository of Dockerized components of the [Pinpoint Application Monitoring](http://naver.github.io/pinpoint/).  
+Official git repository of dockerized components of the [Pinpoint Application Monitoring](http://naver.github.io/pinpoint/).  
 Installing Pinpoint with these docker files will take approximately 10min. to check out the features of pinpoint.
 
 ## What is Pinpoint
 
-[Pinpoint](https://github.com/naver/pinpoint), is the world's leading open-source application monitoring solution - trusted by millions of users around the globe.
-It supports and helps you understand your application in a glance and allow you to build world-class, high-quality software.
+[Pinpoint](https://github.com/naver/pinpoint), is the world's leading open-source application monitoring solution - trusted by millions of users around the world.
```
Comment (RoySRose) [inline Readme.md]: no need of changing to `world`

File: Readme.md [hunk 2]
```diff
@@ -3,81 +3,85 @@
...
+It supports and helps you to understand your applications in a glance and allows you to build world-class, high-quality software.
```
Comment (RoySRose) [inline Readme.md]: no need to include `to`, since it's optional.
Beside the `to`, you've got point

File: Readme.md [hunk 3]
```diff
@@ -3,81 +3,85 @@
...
 
-![Pinpoint](https://github.com/naver/pinpoint-docker/blob/master/docs/Pinpoint-Flink%20upload.png)
- 
+![Pinpoint](https://github.com/naver/pinpoint-docker/blob/master/docs/pinpoint-flink%20upload.png)
+
 ### Testing QuickStart application
- 
-Now you are ready to monitor the sample application(Pinpoint-QuickStart [port 8000](http://localhost:8000)) provided.
+
+Now you are ready to monit the sample application(pinpoint-QuickStart [port 8000](http://localhost:8000)) provided.
```
Comment (RoySRose) [inline Readme.md]: I think, monitor is the proper word here, tell me if I'm wrong.

File: Readme.md [hunk 4]
```diff
@@ -99,37 +103,39 @@ Finally, since agent needs to acknowledge the collector ip. collector ip needs t
 
 ## Configurations
 
-Configuration relies on supplying `docker-compose` with environment variables defined in `.env` file. So it's recommended to change variables only from `.env` file.
+Configurations relay on supplying `docker-compose` with environment variables defined in `.env` file. So it's recommended to change variables only from `.env` file.
```
Comment (RoySRose) [inline Readme.md]: if it needs to change, shouldn't it be `rely`?

File: Readme.md [hunk 5]
```diff
@@ -3,81 +3,85 @@
...
 
 Register a `job` on to pinpoint-flink server.  
-You can build the `job` from the [open-source of Pinpoint](https://github.com/naver/pinpoint), additional guide is [here](https://github.com/naver/pinpoint/blob/master/doc/application-inspector.md#application-inspector).   
+You can build the `job` from the [open-source of Pinpoint](https://github.com/naver/pinpoint), additional guide is [here](https://github.com/naver/pinpoint/blob/master/doc/application-inspector.md#application-inspector).
 or  
 you can simply upload pre-built jar file under pinpoint-flink/build/pinpoint-flink-job-{pinpoint-version}.jar (beware of the version, it should matched with *PINPOINT_VERSION* in .env file)  
 If anyone have solution to put the job file into flink image without doing manually, please let us know.
 
-Pinpoint-Flink server is running on [port 8081](http://localhost:8081/#/submit). From `submit new job` menu
+pinpoint-flink server is running on [port 8081](http://localhost:8081/#/submit). From `submit new job` menu
```
Comment (RoySRose) [inline Readme.md]: I think this one too

File: Readme.md [hunk 6]
```diff
@@ -3,81 +3,85 @@
...
 Submit the jar file with *com.navercorp.pinpoint.flink.StatStreamingVer2Job* in entry class as below image.
 
-![Pinpoint](https://github.com/naver/pinpoint-docker/blob/master/docs/Pinpoint-Flink%20upload.png)
- 
+![Pinpoint](https://github.com/naver/pinpoint-docker/blob/master/docs/pinpoint-flink%20upload.png)
```
Comment (RoySRose) [inline Readme.md]: If this changes~ the name of the png file needs to be changed. so. no need

File: Readme.md [hunk 7]
```diff
@@ -3,81 +3,85 @@
...
+
 ### Testing QuickStart application
- 
-Now you are ready to monitor the sample application(Pinpoint-QuickStart [port 8000](http://localhost:8000)) provided.
+
+Now you are ready to monitor the sample application(pinpoint-QuickStart [port 8000](http://localhost:8000)) provided.
```
Comment (RoySRose) [inline Readme.md]: I think it will be better to leave all the titles as Capital.

not only the ones I've commented, but all of the titles.

Comment (qudongfang) [reply Readme.md]: I have reset them all and just change the command line which cannot exec.
Even though, I though that we should let all the names equal to the directory/project names.

File: Readme.md [hunk 8]
```diff
@@ -25,7 +29,7 @@ To monitor your agent see [configuration part](#configurations) for further deta
 
 ```
 git clone https://github.com/naver/pinpoint-docker.git
-cd Pinpoint-Docker
```
Comment (qudongfang) [inline Readme.md]: If we don't change here. It won't run.

File: Readme.md [hunk 9]
```diff
@@ -17,6 +17,10 @@ It supports and helps you understand your application in a glance and allow you
  - 1.7.3
  - 1.7.2
 
+## Requirements
+
+- [docker 18.02.0+](https://docs.docker.com/compose/compose-file/)
```
Comment (qudongfang) [inline Readme.md]: Compose file version matters.

**Pull request merged**